### DISCUSSION TITLE
Is there a way around the GHB boundary head is less than cell bottom issue?

### DISCUSSION THREAD

**Discussion by AH-CSM** [modflowpy/flopy#1671] [Q&A] [answered]
Hello!
I am trying to run a model that has GHB, UFZ, and RCH packages. However, the depths of my layers are too shallow compared to the GHB head boundary I have specified. I am trying to understand why I am getting an error that the GHB boundary head is less than the cell bottom if the difference in the head is supposed to be calculated by the conductance equation. Maybe I do not fully understand the way the GHB is implemented. I want to simulate how my larger area water table can rise over time if I were to have an input due to irrigation in a smaller area in my model and I thought the UFZ would solve this issue but it hasn't. Any advice?
Thank you in advance!

**Comment by emorway-usgs** [comment, marked as answer]
@AH-CSM, perhaps you've already solved this, but try applying the GHB to a lower layer...a layer with top and bottom elevations that bracket the elevation you are specifying for the boundary condition.  If the irrigation you are applying with the UZF package raises the water table elevation throughout your model, the GHB boundary type will still drain water out of the model from the lower layers that it is applied to.  Maybe I'm not understanding your question.

**Reply by AH-CSM** [reply]
Yes thank you I did thry that and I am now struggling to understand what technically happens for the boundaries once the ghb is no longer applied to the other layers. Does the program then just apply new convergence criteria based on the initial conditions for those upper layers? I don't think I fully understand how the program calculates the equations when boundary conditions aren't applied. I have scoured the documentation I can find and I still don't fully grasp how it works. I feel that understanding the underlying mechanics of this program is way more important than fixing my model right now. Thank you so much for all the education you are providing!

**Comment by yanyy2019** [comment]
您的信已收到,谢谢！Receiving your reply, thank you!
